### PR TITLE
Allow groupchat hosts to rename their groupchats

### PR DESCRIPTION
**Pull request by AnnikaCodes**

### CODE REVIEW

**Comment by Zarel** [comment]
Should parent roomdrivers be able to rename a subroom groupchat?

File: server/chat-commands/room-settings.ts [hunk 1]
```diff
@@ -933,7 +933,7 @@ export const commands: ChatCommands = {
 		if (room.persist && isGroupchat) return this.errorReply(`This isn't a groupchat.`);
 		if (!room.persist && !isGroupchat) return this.errorReply(`Use /renamegroupchat instead.`);
 		if (isGroupchat) {
-			if (!user.can('lock')) this.checkCan('editroom', null, room);
+			this.checkCan('mute', null, room);
```
Comment (Zarel) [inline server/chat-commands/room-settings.ts:936]: I suppose this _is_ simpler and usually correct, but I can't say I'm a fan of abusing unrelated permissions in the general case.

Comment (AnnikaCodes) [reply server/chat-commands/room-settings.ts:936]: We could use `Auth#atLeast`, I guess? I think that a `renamegroupchat` permission is a bit specific, personally.

**Comment by Zarel** [comment]
I'm not going to block this on the `mute` trick, but this question _is_ relevant:

> Should parent roomdrivers be able to rename a subroom groupchat?

Because I believe this changes that from "no" to "yes", which was probably unintentional.

**Comment by AnnikaCodes** [comment]
Yes, that was unintentional

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**